NethermindNode.Tests/Tests/Receipts/ReceiptsVerification.cs

```diff
@@ -0,0 +1,226 @@
+using NethermindNode.Core.Helpers;
+using Nethereum.Web3;
+using Nethereum.RPC.Eth.DTOs;
+using Nethereum.Hex.HexTypes;
+using System.Diagnostics;
+using NUnit.Framework.Internal;
+using Nethereum.JsonRpc.Client.Streaming;
+using Nethereum.JsonRpc.WebSocketStreamingClient;
+using Nethereum.RPC.Eth.Subscriptions;
+using Newtonsoft.Json;
+using System.Numerics;
+namespace NethermindNode.Tests.Receipts;
+
+class ReceiptsVerification
+{
+  private const string RpcAddress = "http://localhost:8545";
+  private const string WsAddress = "ws://localhost:8545";
+  private Web3 w3 = new Web3(RpcAddress);
+  private static readonly NLog.Logger Logger = NLog.LogManager.GetLogger(TestContext.CurrentContext.Test.Name);
+  public Queue<Block> blocks = new Queue<Block>();
+
+  [Test]
+  [Category("ReceiptsNew")]
+  public async Task Verify_New_Receipts()
+  {
+    var testRunTime = 2 * 60 * 1000; // 10 minutes
```

Comment:
Instead of testRuntime can we check specific amount of blocks? I'd go a bit longer and check at least 128 blocks to ensure flushing happened. So maybe even better to use 200 blocks.

Reply:
Yeah, good point. I've updated the test to process 256 blocks, and then re-verify them